### ISSUE TITLE
Access iCloud files 

### ISSUE THREAD

**Issue by ifuchs** [ColdGrub1384/Pyto#178]
How do I read/write files on my iCloud drive?

**Comment by ColdGrub1384** [comment]
![3DB5993B-E6B2-4130-97F4-39A913946210](https://user-images.githubusercontent.com/19255527/81596177-9ce4c200-9391-11ea-8e4f-20497a786ff2.gif)

**Comment by ifuchs** [comment]
My question was incomplete if I create an iCloud file in, for example,
Shortcuts, is there a way I can access this file from Pyto? If so, what is
the path?

**Comment by ColdGrub1384** [comment]
If you want to write to an app’s folder, you have to select it. It’s not in the same directory than other folders in iCloud Drive.

**Comment by ifuchs** [comment]
Do I have to select it in files and move it to the Pyto folder? If not,
what path would be used to open a file in the Shortcuts folder?

**Comment by ColdGrub1384** [comment]
No, you can change the current directory of the script by pressing the gear icon.
